package.json

```diff
@@ -7,7 +7,11 @@
   "scripts": {
     "build": "rm -rf dist && rollup --config",
     "lint": "eslint .",
-    "prepublishOnly": "npm run lint && npm test && npm run build && npm run test:bundle && npm run test:e2e",
+    "prepublishOnly": "echo \"\nPlease use 'npm run release' instead\n\" && exit 1",
```

Comment:
Are you doing this for backward compatibility? Wondering why we need this.

Reply:
I added this incase someone tries to run `npm publish` in this repo. This will prevent that operation and recommend `npm run release` instead.

Reply:
What catches someone trying to run `publish` and then fires off `npm run prepublishOnly`?

Reply:
Ah yes, that's built into the npm publish command workflow itself. Here's the definition of prepublishOnly:

> prepublishOnly: Run BEFORE the package is prepared and packed, ONLY on npm publish

https://docs.npmjs.com/misc/scripts

Reply:
Ah, figured it might be something like that.  Thanks for clarifying.